server/src/main/java/org/mixit/cesar/cfp/model/ProposalVote.java

```diff
@@ -0,0 +1,77 @@
+package org.mixit.cesar.cfp.model;
+
+import com.fasterxml.jackson.annotation.JsonView;
+import org.mixit.cesar.site.model.FlatView;
+import org.mixit.cesar.site.model.member.Staff;
+
+import javax.persistence.*;
+import java.util.Objects;
+
+@Entity
+@Table(uniqueConstraints={@UniqueConstraint(columnNames={"proposal_id", "voter_id"})})
+public class ProposalVote {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @JsonView(FlatView.class)
+    private Long id;
+
```

Comment:
I don't like in UX when you can make the same action in several screens. You can display the button in the list to know how we had voted. What do you mean of that ?